.azure-pipelines/download-openapidocs.yml

```diff
@@ -0,0 +1,112 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Generates a release build artifact (nuget) from HEAD of master for auth module.
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+pool:
+  vmImage: "windows-latest"
+
+variables:
+  BRANCH: 'weeklyOpenApiDocsDownload'
+
+schedules:
+    - cron: "0 12 * * 5" # Run Every Friday at Midnight UTC Time
```

Comment:
For it to trigger at Midnight UTC, the [cron](https://docs.microsoft.com/en-us/azure/devops/pipelines/process/scheduled-triggers?view=azure-devops&tabs=yaml#supported-cron-syntax) has to be `0 0 * * 5`. 

@MIchaelMainer At what time does the metadata get updated? We would ideally want to sync this after the metadata has been update. Maybe we can run it a few hours after the CSDL captured? @ddyett, @darrelmiller, thoughts?

Reply:
Tuesday morning capture. If this is run on Wednesday, it gives us time to unblock ourselves via transform.

Reply:
We should also investigate if we can trigger on a commit to the onboarding repo.  The problem with using the timer based trigger is when the AGS team do a rollback we miss it.

Reply:
Does the DevX API request a new CSDL file from AGS when we make a request for OpenAPI spec? Where and when does the DevX API get the CSDL? 

To be clear, I have no idea how the CSDL gets to production. I'd like to know how that happens.

Reply:
@darrelmiller [web hooks](https://docs.microsoft.com/en-us/azure/devops/service-hooks/services/webhooks?view=azure-devops) might be a viable solution.

Reply:
Due to changes and accommodations in the DevX API, having a Wednesday schedule seems satisfactory to get the latest OpenApiDocs.